spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -19,12 +19,12 @@
   it "will perform a full refresh on localhost" do
     VCR.use_cassette(described_class.name.underscore.to_s,
                      :allow_unused_http_interactions => true,
+                     :match_requests_on              => [:method, VCR.request_matchers.uri_without_params(:start)],
```

Comment:
Is this needed because of the new inventory?
I don't understand why the query params may be changing in each request.

Reply:
The `start` param is a timestamp, it's used to query the latest metric for availability.